staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -393,6 +394,28 @@ impl VirtioGpu {
         Ok(OkNoData)
     }
 
+    /// Moves the cursor's position to the given coordinates.
+    pub fn move_cursor(
+        &mut self,
+        resource_id: u32,
+        gpu_backend: &mut GpuBackend,
+        cursor: VhostUserGpuCursorPos,
+    ) -> VirtioGpuResult {
+        if resource_id == 0 {
```

Comment:
This doesn't seem to follow the spec: 
> **VIRTIO_GPU_CMD_MOVE_CURSOR** Move cursor. Request data is struct virtio_gpu_update_cursor. Re-
sponse type is VIRTIO_GPU_RESP_OK_NODATA.
Move cursor to the place specified in pos. The other fields are not used and will be ignored by the
device.

Specifically the `The other fields are not used and will be ignored by the device.` is not followed.  You shouldn't check the `resource_id` here and call `cursor_pos_hide`. 


https://docs.oasis-open.org/virtio/virtio/v1.2/csd01/virtio-v1.2-csd01.pdf